doc/api/child_process.markdown

```diff
@@ -569,18 +305,19 @@ Example of checking for failed exec:
 
 #### options.detached
 
-On Windows, this makes it possible for the child to continue running after the
-parent exits. The child will have a new console window (this cannot be
-disabled).
+On Windows, setting `options.detached` to `true` makes it possible for the
+child process to continue running after the parent exits. The child will have
+its own console window. *Once for a child process, this cannot be disabled*.
 
-On non-Windows, if the `detached` option is set, the child process will be made
-the leader of a new process group and session. Note that child processes may
-continue running after the parent exits whether they are detached or not.  See
-`setsid(2)` for more information.
+On non-Windows platforms, if `options.detached` is set to *true*, the child
+process will be made the leader of a new process group and session. Note that
+child processes may continue running after the parent exits regardless of
+whether they are detached or not.  See `setsid(2)` for more information.
 
-By default, the parent will wait for the detached child to exit.  To prevent
-the parent from waiting for a given `child`, use the `child.unref()` method,
-and the parent's event loop will not include the child in its reference count.
+By default, the parent will wait for the detached child to exit. To prevent
+the parent from waiting for a given `child`, use the `child.unref()` method.
+Doing so will cause the parent's event loop to not include the child in its
+reference count, allowing the parent to exit independently of the child.
```

Comment:
Only sometimes. If you have an IPC channel (spawn with ipc, or fork), you also have to unref the control channel using an undocumented property. We should document that property.